processing-service-core/api/swagger/swagger.yaml

```diff
@@ -952,7 +954,123 @@ paths:
           description: The specified Workflow could not be found.
           schema:
             $ref: "#/definitions/errorMessage"    
+  ##########################
+  ##  Default Config Store
+  ##########################
+  /globalConfig/{key}:
+    x-swagger-router-controller: globalConfig
+    parameters:
+      - name: key
+        in: path
+        type: string
+        description: The identifier of global config for tenant.
+        required: true
+    put:
+      tags:
+        - Global Config
+      parameters:
+        - name: default
+          type: string
+          in: query
+          description: The value to use when setting the global config for the tenants key.
+          required: true
+        - name: description
+          type: string
+          in: query
+          description: The description of the global configuration value.
+          #required: false
+      summary: Creates global config for the key.
+      description: Creates a global configuration setting for this key or updates the currently existing global configuration setting if it already exists.
+      operationId: createGlobalConfig
```

Comment:
Change to 'setGlobalConfig' because this can either create or update global config.